management_api.go

```diff
@@ -14,7 +14,13 @@
 
 package casbin
 
-import "github.com/Knetic/govaluate"
+import (
+	"errors"
+	"fmt"
+	"github.com/Knetic/govaluate"
+	"github.com/casbin/casbin/v2/util"
+	"strings"
```

Comment:
sort

Reply:
Seems to be because the uppercase ``K`` in ``github.com/Knetic/govaluate`` is ordered before the lowercase ``c`` in ``github.com/casbin/casbin/v2/util``

Reply:
@tangyang9464 use this: https://github.com/golang/go/issues/37719#:~:text=Hello%2C%20world!%22))%0A%7D-,goimports,-(note%20the%20imports

![image](https://user-images.githubusercontent.com/3787410/168069069-cc4fbce6-3a36-4b72-ad62-d973811e29d2.png)

Reply:
got it